api/src/main/java/org/openmrs/module/fhir2/providers/r3/LocationFhirResourceProvider.java

```diff
@@ -57,6 +61,38 @@ public Location getLocationById(@IdParam @NotNull IdType id) {
 		return Location30_40.convertLocation(location);
 	}
 	
+	
+	@Create
+	public MethodOutcome createLocation(@ResourceParam Location location) {
+		return FhirProviderUtils.buildCreate(locationService.create(Location30_40.convertLocation(location)));
+	} 
+
+	@Update
+	@SuppressWarnings("unused")
+	public MethodOutcome updateLocation(@IdParam IdType id, @ResourceParam Location location) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update");
+		}
+
+		location.setId(id.getIdPart());
+
+		return FhirProviderUtils.buildUpdate(locationService.update(id.getIdPart(),
+		    Location30_40.convertLocation(location)));
+	}
+
+	@Delete
+	@SuppressWarnings("unused")
+	public OperationOutcome deleteLocation(@IdParam @NotNull IdType id) {
+		org.hl7.fhir.r4.model.Location location = locationService.delete(id.getIdPart());
```

Comment:
@gitcliff  did you try importing org.hl7.fhir.r4 package and see the reaction?

Reply:
no i dint want to import it while working in r3